### PR TITLE
Rebrands the password reset screen

### PR DESCRIPTION
**Pull request by devkonrad**
This PR rebrands the Reset Password Screen.

Captura de tela de 2022-08-23 08-13-28

Refs.: #3578

### CODE REVIEW

**Comment by dboyer** [comment]
@devkonrad Looks good! Could you check if the font size on the button text is the same font size as the button text on the sign in and register pages? I'd like them to be consistent.

**Comment by devkonrad** [comment]
> @devkonrad Looks good! Could you check if the font size on the button text is the same font size as the button text on the sign in and register pages? I'd like them to be consistent.

Hi, @dboyer! I had the impression that the button font size was bigger than the sign up, too. But i used the exact same class, I guess that its a optcal illusion because here we have more text than the sign up screen.

**Comment by dboyer** [comment]
Interesting! That makes sense that the amount of text could fool the eye. Thanks for checking!

**Review by viviancan — CHANGES_REQUESTED** [review]
Hey Alex! Thx for tackling this one! I am leaving one comment on the ticket about the font for the "Email" label! Besides that, it looks good!

**Comment by viviancan** [comment]
The "email" label font should match the updated font like on the sign-in page. I think this is our old branding font :) 
CleanShot 2022-08-24 at 09 54 28@2x

**Comment by devkonrad** [comment]
> @devkonrad The "email" label font should match the updated font like on the sign-in page. I think this is our old branding font :) CleanShot 2022-08-24 at 09 54 28@2x
> 
> Example below CleanShot 2022-08-24 at 10 01 25@2x

Good caught, @viviancan ! I will fix it! ;)

File: app/views/password_resets/new.html.erb [hunk 1]
```diff
@@ -1,15 +1,16 @@
 <% provide :title, "Reset your password" %>
+<% provide :main_header, render('application/templates/tg_green_header', heading: "Welcome Back") %>
```
Comment (shaunxp20) [inline app/views/password_resets/new.html.erb:2]: I tried this one locally and I was able to reset a password! 👍 The one thing popping to me is this header "Welcome Back", maybe that should say something else for the password reset page? 🤔  @dboyer

Comment (dboyer) [reply app/views/password_resets/new.html.erb:2]: Hmmmm.......good question. It's not the best fit but I'm also not coming up with anything better so let's keep it as "welcome back" for now. Probably if they are resetting a password they've been on the site at least once so it's not _wrong_ to welcome them back, just not expected.

**Review by shaunxp20 — APPROVED** [review]

**Comment by devkonrad** [comment]
> > @devkonrad The "email" label font should match the updated font like on the sign-in page. I think this is our old branding font :) CleanShot 2022-08-24 at 09 54 28@2x
> > Example below CleanShot 2022-08-24 at 10 01 25@2x
> 
> Good caught, @viviancan ! I will fix it! ;)

Fixed it yet, @viviancan! :)

**Pull request merged**